### PR TITLE
Terminology Update: .NET platform -> .NET implementation

### PR DESCRIPTION
**Pull request by tompratt-AQ**
## Summary
Of the 65 instances of ".NET platform(s)" to ".NET implementation(s)" in 35 files. I changed 60 instances in 32 files. I also changed all instances of ""Core CLR" to "CoreCLR". 

Some of the changes will be done as part of a commit to PR 2818, as they involve the same files.

## Suggested Reviewers
@rpetrusha

### CODE REVIEW

**Comment by tompratt-AQ** [comment]
@rpetrusha I resolved the conflicting file using the "Resolve Conflicts" button, and it indicated that the conflict was resolved. So, I'm not sure why the message still appears.

**Comment by rpetrusha** [comment]
@tompratt-AQ, I've fixed it.

File: docs/standard/choosing-core-framework-server.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ ms.assetid: 155553e4-89a2-418d-be88-4e75f6c3cc69
 
 # Choosing between .NET Core and .NET Framework for server apps
 
-There are two supported choices of runtime for building server-side applications with .NET: .NET Framework and .NET Core. Both share a lot of the same .NET platform components and you can share code across the two. However, there are fundamental differences between the two and your choice will depend on what you want to accomplish.  This article provides guidance on when to use each.
+There are two supported implementations for building server-side applications with .NET: .NET Framework and .NET Core. Both share a lot of the same .NET Standard components and you can share code across the two. However, there are fundamental differences between the two and your choice will depend on what you want to accomplish.  This article provides guidance on when to use each.
```
Comment (rpetrusha) [inline docs/standard/choosing-core-framework-server.md]: same .NET Standard components --> same components

File: docs/standard/class-libraries.md [hunk 2]
```diff
@@ -42,7 +42,7 @@ Many library developers have switched from producing multiple platform-specific
```
Comment (rpetrusha) [inline docs/standard/class-libraries.md]: The heading should read ".NET Standard class libraries", not ".NET Core Class Libraries"

File: docs/standard/class-libraries.md [hunk 3]
```diff
@@ -42,7 +42,7 @@ Many library developers have switched from producing multiple platform-specific
 
 .NET Core libraries are a replacement of the platform-specific and portable libraries concepts. They are platform-specific in the sense that they expose all functionality from the underlying platform (no synthetic platforms or platform intersections). They are portable in the sense that they work on all supporting platforms.
 
-.NET Core exposes a set of library _contracts_. .NET platforms must support each contract fully or not at all. Each platform, therefore, supports a set of .NET Core contracts. The corollary is that each .NET Core class library is supported on the platforms that support it’s contract dependencies.
+.NET Core exposes a set of library _contracts_. .NET implementations must support each contract fully or not at all. Each implementation, therefore, supports a set of .NET Core contracts. The corollary is that each .NET Core class library is supported on the platforms that support it’s contract dependencies.
```
Comment (rpetrusha) [inline docs/standard/class-libraries.md]: .NET Core exposes --> The .NET Standard 
.NET Core contracts --> .NET Standard contracts
.NET Core class library --> .NET Standard class library

File: docs/standard/class-libraries.md [hunk 4]
```diff
@@ -42,7 +42,7 @@ Many library developers have switched from producing multiple platform-specific
...
 
 .NET Core contracts do not expose the entire functionality of the .NET Framework (nor is that a goal), however, they do expose many more APIs than Portable Class Libraries. More APIs will be added over time.
```
Comment (rpetrusha) [inline docs/standard/class-libraries.md]: .NET Core contracts do --> The .NET Standard does

File: xml/System.Timers/Timer.xml [hunk 5]
```diff
@@ -44,7 +44,7 @@
...
   
 > [!IMPORTANT]
->  The <xref:System.Timers.Timer> class is available in the .NET Framework only. It is not included in the .NET Standard and is not available on other platforms, such as .NET Core or the Universal Windows Platform. On these platforms, as well as for portability across all .NET platforms, you should use the <xref:System.Threading.Timer?displayProperty=fullName> class instead.  
+>  The <xref:System.Timers.Timer> class is available in the .NET Framework only. It is not included in the .NET Standard and is not available on other platforms, such as .NET Core or the Universal Windows Platform. On these platforms, as well as for portability across all .NET implementations, you should use the <xref:System.Threading.Timer?displayProperty=fullName> class instead.
```
Comment (rpetrusha) [inline xml/System.Timers/Timer.xml]: There are a couple more occurrences of platform in this text that should be changed to implementation.

File: xml/System/Type.xml [hunk 6]
```diff
@@ -296,7 +296,7 @@
   
  Similarly, if the current <xref:System.Type> object represents an unassigned generic parameter `T`, this property returns the assembly that contains the generic type that defines `T`.  
   
- If the <xref:System.Type.Assembly%2A?displayProperty=fullName> property is not available on a particular .NET platform, such as .NET Core or the Universal Windows Platform, use the  `System.Reflection.TypeInfo.Assembly` property instead. ?qualifyHint=False&autoUpgrade=True  
+ If the <xref:System.Type.Assembly%2A?displayProperty=fullName> property is not available on a particular .NET implementation, such as .NET Core or the Universal Windows Platform, use the  `System.Reflection.TypeInfo.Assembly` property instead. ?qualifyHint=False&autoUpgrade=True
```
Comment (rpetrusha) [inline xml/System/Type.xml]: "use the  `System.Reflection.TypeInfo.Assembly` property instead. ?qualifyHint=False&autoUpgrade=True": I wonder how this happened. it should be `use the <xref:System.Reflection.TypeInfo.Assembly?displayProperty=fullName> property instead.`

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Looks good, @tompratt-AQ. I've left some comments on.

**Comment by rpetrusha** [comment]
Thanks for making the changes, @tompratt-AQ. This looks good, so I'll merge it now.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Actually, take that back, @tompratt-AQ. I'll merge when the build finishes.

**Comment by tompratt-AQ** [comment]
@rpetrusha Thanks Ron!

**Pull request merged**